keychain_pre1.10.go

```diff
@@ -463,6 +474,16 @@ func uintptrCFStringToString(s uintptr) string {
 	return string(buf[:usedBufLen])
 }
 
+// untptrCFDateToTime converts a uintptr (assumed to have been
+// converted from a CFDateRef) to a time.Time.
+//
+// This is an adaptation of CFDateTimeToTime.
+func uintptrCFDateToTime(p uintptr) time.Time {
+	abs := C.CFDateGetAbsoluteTimeSafe(C.uintptr_t(p))
```

Comment:
This function (and `CFDateGetAbsoluteTimeSafe` above) is what's needed to get this working on go 1.9